elotl/utils/util.py

```diff
@@ -0,0 +1,7 @@
+langcode2macrolang = {
```

Comment:
Creo que no me encanta el nombre ni del archivo (`util.py`) ni de la variable. Quizá cambiaría a algo más explicito como:

```suggestion
MACROLANGS = {
```

Considerando que será una variable global usaría mayúsculas

Reply:
Respecto al nombre del archivo creo que no es buena práctica utilizar este tipo de helpers pero podemos dejar su refactorización (si hace sentido) para más adelante (CC: @paoinnesb). Dejo un [recurso que habla al respecto](https://dev.to/knzt/helpers-and-utils-folders-in-software-architecture-3f8h)

Reply:
Vale, tienes razón del nombre y también la cohesión, quizá de momento no sea relevante pensar en estos detalles, pero cuando crezca el código podríamos evaluar si conviene repensar los `utils`.

Quizá solo insistiría en usar las mayúsculas, ya que es una variable global ¿Qué opinas?

```suggestion
LANGCODE2MACROLANG = {
```